tools/inference_gradio.py

```diff
@@ -47,16 +47,15 @@ def get_args() -> Namespace:
 
     Example:
 
-        >>> python tools/inference_gradio.py --model ./anomalib/models/padim/config.yaml --weight_path ./results/padim/mvtec/bottle/weights/model.ckpt
-        >>> python tools/inference_gradio.py --model padim --weight_path ./results/padim/mvtec/bottle/weights/model.ckpt
+        >>> python tools/inference_gradio.py \
+             --config ./anomalib/models/padim/config.yaml \
+             --weight_path ./results/padim/mvtec/bottle/weights/model.ckpt
 
     Returns:
         Namespace: List of arguments.
     """
     parser = ArgumentParser()
-    parser.add_argument(
-        "--model", type=str, required=True, help="Can be either the model name or the path to a model config file"
-    )
+    parser.add_argument("--config", type=Path, required=True, help="Path to a model config file")
     parser.add_argument("--weight_path", type=Path, required=True, help="Path to a model weights")
     parser.add_argument("--meta_data", type=Path, required=False, help="Path to JSON file containing the metadata.")
```

Comment:
there is a bit of inconsistency in naming. In fact, this is the case for other entrypoints. I think we should stick to one of the following

```
- config, weights, meta_data
- config_path, weight_path, meta_data_path
```

Reply:
That's alright. The whole point of the review is to ensure code quality